config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id", to: "guests#show"
```

Comment:
As mentioned above, consider: `post "/guests/:guest_id/rooms", to "guest_rooms#create"`